lumicks/pylake/tests/test_import_time.py

```diff
@@ -0,0 +1,21 @@
+from textwrap import dedent
+import numpy as np
+import subprocess
+import os
+
+
+def test_disabling_capturing(capsys):
+    repeats = 3
+
+    code = dedent("""\
+        import time
+        tic = time.time()
+        import lumicks.pylake
+        print(time.time() - tic)
+        """)
+
+    times = [float(subprocess.check_output(['python', '-c', code])) for i in range(repeats)]
```

Comment:
Use `sys.executable` to get the same Python as you're running in now. Otherwise, you play the OS lottery of what Python (if any) you'll get.
```suggestion
    times = [float(subprocess.check_output([f'{sys.executable}', '-c', code])) for i in range(repeats)]
```

Reply:
👍